chili-epfl/FROG
# Added logout feature to Wiki

This PR introduces the ability for a user to logout from the wiki. 

**How to test?**

- Open the wiki and login as any user.
- Click logout and you should be redirected to the FROG user not logged in page.

**Asana Issue** 

https://app.asana.com/0/1121331595459324/1125930882647545/f
